src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -118,6 +123,12 @@
      */
     public boolean isActive = true;
 
+    /**
+     * Whether to snooze the active or upcoming trip. This is reset upon advancing to the next possible trip date in the
+     * CheckMonitoredTrip job.
+     */
+    public boolean snoozed = false;
```

Comment:
Perhaps this should instead be an integer that can get set to snooze multiple days at a time?

Reply:
That seems more complex than just snoozing for the day and would require a corresponding UI refactor.